daemon/list.go

```diff
@@ -72,9 +72,17 @@ type listContext struct {
 	// exitAllowed is a list of exit codes allowed to filter with
 	exitAllowed []int
 	// beforeContainer is a filter to ignore containers that appear before the one given
+	// Deprecated, this is used for --before=
 	beforeContainer *Container
 	// sinceContainer is a filter to stop the filtering when the iterator arrive to the given container
+	// Deprecated, this is used for --since=
 	sinceContainer *Container
+	// beforeFilter is a filter to ignore containers that appear before the one given
+	// this is used for --filter=before=
+	beforeFilter *Container
```

Comment:
I'm wondering about duplicating those ones, as they serve the exact same purpose, I would tend to think the `beforeContainer` could have been rename `beforeFilter` and the old flag (`--before`) would populate this one too.